geometry_msgs/Pose2D  pose
float32 width